canmatrix/xls.py

```diff
@@ -538,9 +539,9 @@ def load(file, **options):
             (mini, maxi) = test.strip().split("..", 2)
             unit = ""
             try:
-                newSig.offset = float(mini)
-                newSig.min = float(mini)
-                newSig.max = float(maxi)
+                newSig.offset = float_factory(mini)
```

Comment:
I didn't explore if it was actually wrong, but this looks not obviously correct at least.  The assignment of min to offset, that is.

Reply:
this is a mistake - quite sure